Items/07_Items.txt

```diff
@@ -205,6 +205,80 @@ Road Flare ($15):
 light for 5m in every direction for about an hour. Burns in rain, mud, even 
 burns underwater. Don't touch the end or you'll take 10 fire damage.
 
+Datapad (130$):
+	A handheld device with a screen and keypad. Used to record
+information up to 1TB. 
+
+Binoculars (90$):
+	A bifocal optical instrument with a lens for each eye, 
+used for viewing distant objects up to 2km. 
+
+Industrial Paint (35$):
+	A metal canister of wear-resistant paint. Can be used 
+with a brush or similar implement to cover up to 20 square meters of
+flooring, wall or other surface. Paint comes in the following colors.
+
+* Black
+* White
+* Red
+* Blue
+* Green
+* Yellow
+* Orange
+* Purple
+* Brown
+
+Adhesive Foam (45$):
+	A spray can with a nozzle on the end that sprays a chemical
+solution in either an area or a thin line. As the solution dries, 
+it expands and hardens to accomodate the space given. Can be used
+to insulate an area, temporarily patch up a damaged wall, 
+seal ducts, etc. One can has 20 square meter's worth of usage.
+
+Solvent (45$):
+	A spray can with a nozzle on the end that sprays a chemical
+solution in an area. When this solution comes into contact with
+adhesive foam or similar substances, they will dissolve into a vapor.
+One can has 20 square meter's worth of usage.
+
+Basic Storage Crate (125$):
+	A metal crate secured with an electronic keypad. Can hold
+up to 150lbs. The keypad can be opened with either the correct numeric
+password, a DC 85 Strength skill check or with a DC 75 hacking 
+skill check.
```

Comment:
We may want to know volume or the crate dimensions too.
n-b: Can haz metric system pls?